app/http.php

```diff
@@ -762,12 +764,13 @@ function (string $runtimeId, string $payload, array $variables, int $timeout, st
                 'response' => \mb_strcut($res, 0, 1000000), // Limit to 1MB
                 'stdout' => \mb_strcut($stdout, 0, 1000000), // Limit to 1MB
                 'stderr' => \mb_strcut($stderr, 0, 1000000), // Limit to 1MB
-                'duration' => $executionTime + $coldStartTime,
+                'duration' => $duration + $coldStartDuration,
```

Comment:
We need to change the way we calculate the `$coldStartTime` as someone could pass a negative number in case of custom runtimes and make the duration 0

Reply:
Answered in comment above, coldStartTime can be trusted.

Reply:
👍